### PR TITLE
add missing UI documents

### PR DESCRIPTION
**Pull request by zilongshanren**

### CODE REVIEW

**Comment by zilongshanren** [comment]
@pandamicro  please review this PR.

File: cocos/ui/UIRadioButton.h [hunk 1]
```diff
@@ -136,136 +170,204 @@ class CC_GUI_DLL RadioButton : public AbstractCheckButton
...
      */
     static RadioButtonGroup* create();
     
-    /**
+    /**@~english
      * Add a callback function which would be called when radio button is selected or unselected.
-     *@param callback A std::function with type @see `ccRadioButtonGroupCallback`
+     * @~chinese 
+     * 添加一个回调函数, 它将在单选按钮被选中或者取消的时候被调用.
+     *@param callback @~english A std::function with type @see `ccRadioButtonGroupCallback`
```
Comment (pandamicro) [inline cocos/ui/UIRadioButton.h:233]: Add chinese doc for this parameter

File: cocos/ui/UIRadioButton.h [hunk 2]
```diff
@@ -59,42 +64,61 @@ class CC_GUI_DLL RadioButton : public AbstractCheckButton
...
+     * @~chinese 背景纹理名称。
+     * @param backGroundSelected    @~english backGround selected state texture.
+     * @~chinese 背景纹理选择状态纹理名称。
+     * @param cross    @~english cross texture.
+     * @~chinese 前景纹理名称。
+     * @param backGroundDisabled    @~english backGround disabled state texture.
+     * @~chinese 背景纹理禁用状态名称。
+     * @param frontCrossDisabled    @~english cross dark state texture.
+     * @~chinese 前景禁用状态纹理名称。
      * @param texType    @see `Widget::TextureResType`
```
Comment (pandamicro) [inline cocos/ui/UIRadioButton.h:118]: Not documented

File: cocos/ui/UIRadioButton.h [hunk 3]
```diff
@@ -103,21 +127,31 @@ class CC_GUI_DLL RadioButton : public AbstractCheckButton
...
                             TextureResType texType = TextureResType::LOCAL);
     
-    /**
+    /**@~english
      * Add a callback function which would be called when radio button is selected or unselected.
-     *@param callback A std::function with type @see `ccRadioButtonCallback`
+     * @~chinese 
+     * 注册一个回调函数, 它将在单选按钮被选中或者取消的时候被调用.
+     *@param callback @~english A std::function with type @see `ccRadioButtonCallback`
+     * @~chinese std::函数型
```
Comment (pandamicro) [inline cocos/ui/UIRadioButton.h]: Not understandable

File: cocos/ui/UIRadioButton.h [hunk 4]
```diff
@@ -136,136 +170,205 @@ class CC_GUI_DLL RadioButton : public AbstractCheckButton
...
     
-    /**
+    /**@~english
      * Add a callback function which would be called when radio button is selected or unselected.
-     *@param callback A std::function with type @see `ccRadioButtonGroupCallback`
+     * @~chinese 
+     * 添加一个回调函数, 它将在单选按钮被选中或者取消的时候被调用.
+     *@param callback @~english A std::function with type @see `ccRadioButtonGroupCallback`
+     * @~chinese 类型为 ccRadioButtonGroupCallback的回调函数
+     * @~chinese std::function
```
Comment (pandamicro) [inline cocos/ui/UIRadioButton.h]: Remove this line

**Comment by pandamicro** [comment]
And please squash your commit

**Pull request merged**